Gemfile

```diff
@@ -15,5 +32,7 @@ group :development do
 end
 
 group :plugins do
-  gem 'vagrant-proxyconf', path: File.dirname(__FILE__)
+  gem 'vagrant-proxyconf', path: '.'
+  gem 'vagrant-vbguest', '~> 0.17'
+  gem 'vagrant-omnibus', '1.5.0'
```

Comment:
Do we really need to add dependency to external plugins? Those and their transitive dependencies might also affect the Travis builds, which have been sadly quite fragile at times.

Reply:
Hmm, another good question. I think I ended up adding this because when I was testing "chef proxy" using the chef_solo provider it required `vagrant-omnibus` and since I don't use chef I didn't have that plugin installed in Vagrant so I added it there in order to finish testing the chef proxy provider.  I no longer got errors when testing `bundle exec vagrant provision` after adding `vagrant-omnibus` to the Gemfile and after also enabling the chef provider in the Vagrantfile.

Reply:
I've removed the extra stuff from the Gemfile as requested.